modules/tinymce/src/core/test/ts/browser/EditorTest.ts

```diff
@@ -10,31 +10,29 @@ import * as HtmlUtils from '../module/test/HtmlUtils';
 import URI from 'tinymce/core/api/util/URI';
 import Theme from 'tinymce/themes/silver/Theme';
 
-UnitTest.asynctest('browser.tinymce.core.EditorTest', function (success, failure) {
-  const suite = LegacyUnit.createSuite<Editor>();
-
+UnitTest.asynctest('browser.tinymce.core.EditorTest', (success, failure) => {
   Theme();
 
-  suite.test('Event: change', function (editor) {
+  const sEventChange = (editor: Editor) => Log.step('', 'Event: change', Step.sync(() => {
```

Comment:
Can you please add `TBA` here instead of an empty string? Another option would be to use `Step.label` instead which doesn't require a test id.